src/components/Subscription.js

```diff
@@ -0,0 +1,46 @@
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+
+const SignupSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required('Required'),
+});
+
+const Subscription = () => {
+  return (
+    <div className="mt-[100px] px-3 flex flex-col">
+      <h1 className="text-[#FFFF] mb-2 text-sm">Subscribe to our Newsletter</h1>
+      <Formik
+        className=""
+        initialValues={{
+          email: '',
+        }}
+        validationSchema={SignupSchema}
+        onSubmit={() => {
+          // same shape as initial values
+        }}
+      >
+        {({ errors, touched }) => (
+          <Form className="flex">
```

Comment:
We need this form to be a bit bigger just like it was in Figma. The Error message needs some styling, it kind of looks ugly. And when we click on submit, we need to see a notification saying that users email is added to our newsletter.